pkg/frontend/admin_openshiftcluster_vmresize_test.go

```diff
@@ -110,13 +88,13 @@ func TestAdminVMResize(t *testing.T) {
 					},
 				})
 			},
-			mocks:          func(tt *test, a *mock_adminactions.MockAzureActions, k *mock_adminactions.MockKubeActions) {},
+			mocks:          func(tt *test, a *mock_adminactions.MockAzureActions) {},
```

Comment:
did you know that you can use `nil` here instead? (not really asking to change because the we should not use functions as test variables, but this is for another pr imho) (btw could you open it if you have 1 or 2 spare cycles please :relaxed: {at least for the `mock` functions})

Reply:
Sure, will create a new PR for this. thanks.